vertical-slice-architecture/README.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Vertical Slice Architecture
+aka: Layer By Feature
```

Comment:
Let's leave this line out since it's not in line with the pattern template we use

https://github.com/iluwatar/java-design-patterns/wiki/02.-Pattern-template

Reply:
We can put the alternative name in its own chapter "Also known as"